frontend/javascripts/oxalis/geometries/materials/plane_material_factory.js

```diff
@@ -678,24 +689,24 @@ class PlaneMaterialFactory {
       maximumLayerCountToRender,
     } = Store.getState().temporaryConfiguration.gpuSetup;
 
-    const segmentationLayer = Model.getSegmentationLayer();
+    // Don't compile code for segmentation in arbitrary mode
+    const hasSegmentation = this.isOrthogonal && Model.hasSegmentationLayer();
```

Comment:
Maybe rename to `shouldRenderSegmentation" or so to make the naming and intention more clear.